CMakeLists.txt

```diff
@@ -111,7 +111,17 @@ else()
         add_definitions(-DUSE_PROTOBUF)
 endif()
 
-set(LIBTOFI_LIBDIR_PATH "${CMAKE_SOURCE_DIR}/../libs")
+message("${CMAKE_BUILD_TYPE}")
+
+if(WIN32)
```

Comment:
What happens if we run this on linux systems?

Reply:
Added behaviour for linux